components/server/src/projects/projects-service.ts

```diff
@@ -391,14 +391,21 @@ export class ProjectsService {
                 throw new ApplicationError(ErrorCodes.BAD_REQUEST, "Project name must not be empty.");
             }
         }
-        const allowedFields: (keyof Project)[] = ["settings", "name"];
```

Comment:
@filiptronicek, why is this filter removed?

Reply:
Thanks for pointing this out, @AlexTugarev 🙏. This is an oversight from changing the merging logic. I will bring it back in a follow-up